lmdeploy/archs.py

```diff
@@ -39,21 +39,35 @@ def autoget_backend(model_path: str) -> Union[Literal['turbomind', 'pytorch']]:
         is_supported as is_supported_pytorch
 
     pytorch_has, turbomind_has = False, False
+    is_turbomind_installed = True
     try:
         from lmdeploy.turbomind.supported_models import \
             is_supported as is_supported_turbomind
         turbomind_has = is_supported_turbomind(model_path)
     except ImportError:
-        logger.warning(
-            'Lmdeploy with turbomind engine is not installed correctly. '
-            'You may need to install lmdeploy from pypi or build from source '
-            'for turbomind engine.')
+        is_turbomind_installed = False
 
     pytorch_has = is_supported_pytorch(model_path)
 
-    if not (pytorch_has or turbomind_has):
-        logger.warning(f'{model_path} is not explicitly supported by lmdeploy.'
-                       f' Try to run with lmdeploy pytorch engine.')
+    try_run_msg = (f'Try to run with pytorch engine because `{model_path}`'
+                   f' is not explicitly supported by lmdeploy. ')
+    if is_turbomind_installed:
```

Comment:
I think these warning should only appear when user intent to use turbomind (for example, sending a turbomindconfig)

Reply:
The default is using turbomind. Autobackend works only when backendconfig is None or TurbomindConfig.